README.md

```diff
@@ -110,6 +110,22 @@ To use Better Canvas, select your browser below to install the extension.
 
 ## Contributing
 
+### Localization
+
+This extension is using [chrome.i18n](https://developer.chrome.com/docs/extensions/reference/api/i18n) for localization.
+
+#### Add a new language
+
+- Add a new folder in the `_locales` directory with the name of [Locale code](https://developer.chrome.com/docs/extensions/reference/api/i18n#locales).
+- Add a new `messages.json` file in the new folder.
+- Copy the content from `en/messages.json` to the new file.
+- Translate all strings to the new language.
+
+#### Update an existing language
+
+- Open the `messages.json` file in the desired language folder.
+- Update the strings.
+
```

Comment:
this should be below the "Add a new feature" section IMO, because most people who visit this repo are more likely going to want to add a new feature, than extend the language support